### PR TITLE
x11-wm/enlightenment: bump to 0.22.4

### PR DESCRIPTION
**Pull request by juippis**
Due to efl-1.21.0 not being keyworded on other ARCHES than ~amd64 ~x86 I found it easier to just drop keywords from enlightenment until re-keywording happens. Bug has been opened about it, 
https://bugs.gentoo.org/664482

This is an attempt to get rid of the massive IUSE_MODULES hell that's used in previous versions. All modules can be disabled from settings and building them won't increase compile time by much. 
`enlightenment: Wed Aug 28 08:12:53 2018: 54 seconds`

Oh, I also tried to use BDEPEND for the first time on this one. Hope I did that correctly!

EDIT: I don't want to drop 0.22.3 yet due to changed wayland reqs, this might still be broken.
EDIT-2: Added a USE flag for 'packagekit' module.

### CODE REVIEW

File: x11-wm/enlightenment/enlightenment-0.22.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		x11-libs/libxkbcommon
+		x11-libs/pixman
+	)
+"
+BDEPEND="
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+"
+DEPEND="
+	${BDEPEND}
```
Comment (mgorny) [inline x11-wm/enlightenment/enlightenment-0.22.4.ebuild]: That doesn't seem to make sense. Why would you require those packages inside sysroot? They're only useful to be executed, aren't they?

Comment (juippis) [reply x11-wm/enlightenment/enlightenment-0.22.4.ebuild]: Guess we are all still learning BDEPEND (there are few ebuilds in the tree with similar case). Yes, they're only executed during emerge.

Just change it to `DEPEND="${RDEPEND}"` in here?

Comment (mgorny) [reply x11-wm/enlightenment/enlightenment-0.22.4.ebuild]: Yes.

**Comment by mgorny** [comment]
I'm going to merge it, thanks!

**Pull request closed without merging**